pkg/server/api/bundle/v1/service_test.go

```diff
@@ -1240,6 +1241,236 @@ func TestBatchCreateFederatedBundle(t *testing.T) {
 	}
 }
 
+func TestBatchSetFederatedBundle(t *testing.T) {
+	test := setupServiceTest(t)
```

Comment:
I don't think this is used outside the loop. Maybe initialize inside of the loop so that it doesn't have to be "reset" each time?